src/components/note/__internal__/note.style.js

```diff
@@ -0,0 +1,152 @@
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+import baseTheme from '../../../style/themes/base';
+
+const NoteContent = styled.div`
```

Comment:
On every other component I've looked at so far all the styled components are prefixed with `Styled`, i.e. here would be `StyledNoteContent` - that seems like a good idea so it's immediately obvious when looking at the component file, which bits are styled components.
I'd say worth changing these to be the same

Reply:
Fair point will update the styled components to be consistent